src/util/newconfig/ConfigConstraints.hpp

```diff
@@ -362,6 +362,9 @@ static constinit NumberValueConstraint<uint16_t> gValidateUint16{
     std::numeric_limits<uint16_t>::min(),
     std::numeric_limits<uint16_t>::max()
 };
+
+// log file size minimum is 1mb, log rotation time minimum is 1hr
+static constinit NumberValueConstraint<uint32_t> gValidateLogs{1, std::numeric_limits<uint32_t>::max()};
```

Comment:
I think the limit for time and size happen to be the same. But I will use 2 constraints.